automation_Test/login.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+
+from appium import webdriver
+import time
+import os
+import unittest
+from time import sleep
+import constants as const
+import actions
+
+class loginTest(unittest.TestCase):
+
+    def setUp(self):
+        # set up appium
+        self.driver = actions.getWebDriver()
+        #self.driver.implicitly_wait(60)
+
+    def tearDown(self):
+        self.driver.quit()
+
+    def test_ui_login_ok(self):
+        server_url = const.K_URL_1
+        user = const.K_USER_1
+        password = const.K_PASSWORD_1
+        ssl = const.K_SELF_SIGNED_1
+
+        actions.doLoginWith(self.driver,server_url,user,password,ssl)
+
+        sleep(3)
```

Comment:
Instead of sleep(3) I will add a code to wait up(2minutes or so), until loading is done. If loading, is gone before 2 minutes, the test carry on